profile/profile.go

```diff
@@ -0,0 +1,31 @@
+package profile
+
+import (
+	"github.com/gogo/protobuf/proto"
+	"github.com/micromdm/micromdm/profile/internal/profileproto"
+)
+
+type Mobileconfig []byte
+
+type Profile struct {
+	Identifier string
```

Comment:
this was actually something that needs to be changed. i had mistakenly though that config profiles, when encrypted, were entirely encrypted. if this was the case we needed to capture the identifier in an out-of-band way to reference (e.g. in blueprints). however i totally spaced that only the `PayloadContent` is encrypted in a Profile, so the `PayloadIdentifier` in a profile is _always_ accessible even for a signed profile (just need to CMS decode it).

the revised protobuf, for now would just be the mobileconfig. _however_ i envision it being extended to have "type" ENUM that tells whether it is a full configuration profile, a profile _payload_ only (e.g. OTA), an "interpreted" profile that should have variables replaced, and other forms. also other metadata might eventually be included like modified date and all that jazz. but again for now it really is just the mobileconfig data (and the e.g. Bolt keys can be inferred from the actual profile data).